File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float cambio;
    do
    {
        cambio = get_float("Cambio: ");
    }
    while (cambio < 0);
    int centavos = round(cambio * 100);

    int contador = 0;
    while (centavos >= 25)
    {
        contador++;
        centavos -= 25;
    }
    while (centavos >= 10)
    {
        contador++;
        centavos -= 10;
    }
    while (centavos >= 5)
    {
        contador++;
        centavos -= 5;
    }
    while (centavos >= 1)
    {
        contador++;
        centavos -= 1;
    }
    printf("%d\n", contador);

}

File: pset6/sentimental/cash/cash.py
from cs50 import get_float


def main():

    cambio = 0
    contador = 0
    while (cambio <= 0):
        cambio = get_float("Cambio: ")

    centavos = int(cambio * 100)

    while (centavos >= 25):
        contador += 1
        centavos -= 25

    while (centavos >= 10):
        contador += 1
        centavos -= 10

    while (centavos >= 5):
        contador += 1
        centavos -= 5

    while (centavos >= 1):
        contador += 1
        centavos -= 1

    print(contador)


if __name__ == "__main__":
    main()
